### PR TITLE
Add Camera to Prosegur

### PR DESCRIPTION
**Pull request by dgomes**
## Proposed change

Document new platform addition

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/76428
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/prosegur.markdown [hunk 1]
```diff
@@ -26,3 +26,8 @@ There is currently support for the following device types within Home Assistant:
 - Camera
 
 {% include integrations/config_flow.md %}
+
+### Service `camera.request_image`
+
+This service will have Prosegur cloud service "Request image" from your local camera. This service should only be called seldomly, as Prosegur tends to throttle this service for long periods of time, resulting in error for both this integration and your Prosegur mobile application.
```
Comment (frenck) [inline source/_integrations/prosegur.markdown]:
```suggestion
### Service `camera.request_image`

This service will have Prosegur cloud service "Request image" from your local camera. This service should only be called seldomly, as Prosegur tends to throttle this service for long periods of time, resulting in errors for both this integration and your Prosegur mobile application.
```

File: source/_integrations/prosegur.markdown [hunk 2]
```diff
@@ -14,6 +15,19 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-Integrate [Prosegur](https://www.prosegur.com/) Smart Alarms into Home Assistant. Prosegur is company providing intruder protection and detection systems to ensure the security of your home in Iberia, South America and RSA. They have a line of connected alarm systems that can be controlled remotely through a cloud service. This integration taps in to the cloud service API providing functionalities such as ARM_HOME, ARM_AWAY and DISARM.
+Integrate [Prosegur](https://www.prosegur.com/) Smart Alarms into Home Assistant. 
+
+Prosegur is company providing intruder protection and detection systems to ensure the security of your home in Iberia, South America and RSA.
```
Comment (frenck) [inline source/_integrations/prosegur.markdown]:
```suggestion
Prosegur is a company providing intruder protection and detection systems to ensure the security of your home in Iberia, South America and RSA. 
```

File: source/_integrations/prosegur.markdown [hunk 3]
```diff
@@ -14,6 +15,19 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-Integrate [Prosegur](https://www.prosegur.com/) Smart Alarms into Home Assistant. Prosegur is company providing intruder protection and detection systems to ensure the security of your home in Iberia, South America and RSA. They have a line of connected alarm systems that can be controlled remotely through a cloud service. This integration taps in to the cloud service API providing functionalities such as ARM_HOME, ARM_AWAY and DISARM.
+Integrate [Prosegur](https://www.prosegur.com/) Smart Alarms into Home Assistant. 
+
+Prosegur is company providing intruder protection and detection systems to ensure the security of your home in Iberia, South America and RSA. 
+
+Prosegur has line of connected alarm systems that can be controlled remotely through a cloud service. This integration taps in to the cloud service API providing functionalities similar to their mobile app.
```
Comment (frenck) [inline source/_integrations/prosegur.markdown]:
```suggestion
Prosegur has a line of connected alarm systems that can be controlled remotely through a cloud service. This integration taps into the cloud service API, providing functionalities similar to their mobile app.
```

File: source/_integrations/prosegur.markdown [hunk 4]
```diff
@@ -14,6 +15,19 @@ ha_platforms:
...
+
+There is currently support for the following device types within Home Assistant:
+- Alarm
+- Camera
 
 {% include integrations/config_flow.md %}
+
+### Service `camera.request_image`
+
+This service will have Prosegur cloud service "Request image" from your local camera. This service should only be called seldom, as Prosegur tends to throttle this service for long periods of time, resulting in error for both this integration and your Prosegur mobile application.
```
Comment (frenck) [inline source/_integrations/prosegur.markdown]:
```suggestion
This service will have Prosegur cloud service "Request image" from your local camera. This service should only be called seldom, as Prosegur tends to throttle this service for long periods of time, resulting in errors for both this integration and your Prosegur mobile application.
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by dgomes** [comment]
Thank you for the corrections!

**Comment by dgomes** [comment]
Parent has been merged :)

**Pull request merged**